org.adempiere.base/src/org/compiere/print/ServerReportCtl.java

```diff
@@ -289,6 +289,9 @@ static public boolean startStandardReport (ProcessInfo pi)
 			MQuery query = MQuery.get (ctx, pi.getAD_PInstance_ID(), TableName);
 			PrintInfo info = new PrintInfo(pi);
 			re = new ReportEngine(ctx, format, query, info, pi.isSummary(), pi.getTransactionName());
+			if (pi.getSerializableObject() != null && pi.getSerializableObject() instanceof MPrintFormat) {
+				re.setPrintFormat((MPrintFormat)pi.getSerializableObject());

```

Comment:
why is this needed here ? the printformat from pi.getSerializableObject() is already pass into the ReportEngine constructor at line 291 above.